src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/DraftReferralTest.kt

```diff
@@ -0,0 +1,64 @@
+package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
+
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.integration.IntegrationTestBase
+import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Referral
+import java.text.SimpleDateFormat
+import java.util.UUID
+
+class DraftReferralTest : IntegrationTestBase() {
+
+  @Test
+  fun `create draft referral dto from empty referral`() {
+    assertThrows<RuntimeException> {
```

Comment:
What do you think about adding the expected outcome into the test, so that the intent is clear from the test outline without looking at the test code?

E.g. `empty input cannot create draft referral`?

Reply:
ok.  I'll update to make it clearer